### DISCUSSION TITLE
UK Stores need updating with latest cards

### DISCUSSION THREAD

**Discussion by canopus-2** [jef/streetmerchant#1348] [General]
## Expected Behavior

All UK stores should be updated with the most up-to-date links, labels and new graphics cards.

## Current Behavior

Currently, the UK stores require an update as a lot of links are invalid, and some adjustments also need to be made with the parsing information (labels) to capture the correct information. Additionally, a specific RTX 3070 card is unable to be parsed on the Scan.co.uk linkbuilder and therefore needs fixing.

## Steps to Reproduce

1. Check some of the UK stores.
2. Notice there are duplicate 3060ti linkbuilders.
3. Notice there are no links or adjustments to account for the RX 6900XT.

## Environment

**OS: Windows 10**

**Comment by canopus-2** [comment]
I've created a PR for this @jef: #1372

**Reply by jef** [reply]
Awesome work! Thank you.
